docs/reference/src/main/asciidoc/extend.asciidoc

```diff
@@ -137,22 +138,29 @@ The observer method may inject a `BeanManager`
 
 An extension observer method is not allowed to inject any other object.
 
+==== Configurators
+
+CDI 2.0 introduced the Configurators API - a new way to easily configure some parts of the SPI during container lifecycle event notification.
+E.g. to add a qualifier to a bean an extension can observe `ProcessBeanAttributes`, then obtain a configurator instance through `ProcessBeanAttributes.configureBeanAttributes()` and finally use `BeanAttributesConfigurator.addQualifier(Annotation)`.
+No need to wrap/delegate to the original `BeanAttributes`.
+See also chapter http://docs.jboss.org/cdi/spec/2.0-PRD/cdi-spec.html#configurators[Configurators interfaces] of the CDI specification.
```

Comment:
Shouldn't this link lead to PFD instead of PRD? Not that it would matter much, since we will need to replace it with final ones eventually.